test/createCatalystDeploymentStream.spec.ts

```diff
@@ -1,176 +1,180 @@
-      {
-        fetcher: components.fetcher,
-        downloadQueue: components.downloadQueue,
-        logs: components.logs,
-        deployer,
-        metrics: components.metrics,
-        storage: components.storage,
-        processedSnapshotStorage: components.processedSnapshotStorage,
-        processedSnapshots: components.processedSnapshots
-      },
-      {
-        contentServer: await components.getBaseUrl(),
-        tmpDownloadFolder: contentFolder,
-        pointerChangesWaitTime: 0,
-        requestRetryWaitTime: 0,
-        requestMaxRetries: 10,
-        reconnectTime: 50,
-        fromTimestamp: 0,
-      }
-    )
-
-    expect(stream.isStopped()).toEqual(true)
-
-    const startPromise = stream.start()
-    while (stream.isStopped()) {
-      await sleep(1)
-    }
-
-    expect(stream.isStopped()).toEqual(false)
-    await deployer.onIdle()
-    await startPromise
-
-    expect({ snapshotHits }).toEqual({ snapshotHits: 1 })
-
-    expect(stream.getRetryCount()).toEqual(1)
-    expect(stream.isStopped()).toEqual(true)
-
-    expect(r).toEqual([
-      { entityType: 'profile', entityId: 'Qm000001', localTimestamp: 1, authChain, pointers: ['0x1'], snapshotHash: downloadedSnapshotFile },
-      { entityType: 'profile', entityId: 'Qm000002', localTimestamp: 2, authChain, pointers: ['0x1'], snapshotHash: downloadedSnapshotFile },
-      { entityType: 'profile', entityId: 'Qm000003', localTimestamp: 3, authChain, pointers: ['0x1'], snapshotHash: downloadedSnapshotFile },
-      { entityType: 'profile', entityId: 'Qm000004', localTimestamp: 4, authChain, pointers: ['0x1'], snapshotHash: downloadedSnapshotFile },
-      { entityType: 'profile', entityId: 'Qm000005', localTimestamp: 5, authChain, pointers: ['0x1'], snapshotHash: downloadedSnapshotFile },
-      { entityType: 'profile', entityId: 'Qm000006', localTimestamp: 6, authChain, pointers: ['0x1'], snapshotHash: downloadedSnapshotFile },
-      { entityType: 'profile', entityId: 'Qm000007', localTimestamp: 7, authChain, pointers: ['0x1'], snapshotHash: downloadedSnapshotFile },
-      { entityType: 'profile', entityId: 'Qm000008', localTimestamp: 8, authChain, pointers: ['0x1'], snapshotHash: downloadedSnapshotFile },
-      { entityType: 'profile', entityId: 'Qm000009', localTimestamp: 9, authChain, pointers: ['0x1'], snapshotHash: downloadedSnapshotFile },
-      { entityType: 'profile', entityId: 'Qm000010', localTimestamp: 10, authChain, pointers: ['0x1'] },
-      { entityType: 'profile', entityId: 'Qm000011', localTimestamp: 11, authChain, pointers: ['0x1'] },
-      { entityType: 'profile', entityId: 'Qm000012', localTimestamp: 12, authChain, pointers: ['0x1'] },
-      { entityType: 'profile', entityId: 'Qm000013', localTimestamp: 13, authChain, pointers: ['0x1'] },
-    ])
-
-    expect(stream.getGreatesProcessedTimestamp()).toEqual(13)
-  })
-})
+
+it('test', () => { })
+// import { createCatalystDeploymentStream } from '../src'
```

Comment:
Would it be better to delete this test?

Reply:
yes, chore stuff and unit/integration testing will be done once the functionality is verified so this tasks are not done multiple times :(

Reply:
file removed.

Reply:
Don't forget this! :)